typing/env.ml

```diff
@@ -1061,7 +1059,7 @@ let rec lookup_module_descr_aux ?loc lid env =
       let {md_type=mty2} = find_module p2 env in
       begin match get_components desc1 with
         Functor_comps f ->
-          Misc.may (!check_modtype_inclusion env mty2 p2) f.fcomp_arg;
+          Misc.may (!check_modtype_inclusion ~loc:(match loc with Some l -> l | None -> Location.none) env mty2 p2) f.fcomp_arg;
```

Comment:
I think this line is too long (and a bit messy). Could you lift the `(match loc ...)` bit out to it's own binding.